docker-autoscale/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN microdnf install unzip
 ADD ${REPO_URL}/com/sequenceiq/periscope/$VERSION/periscope-$VERSION.jar /periscope.jar
 
 # add jmx exporter
-ADD jmx_prometheus_javaagent-0.16.1.jar /jmx_prometheus_javaagent.jar
+ADD jmx_prometheus_javaagent-0.17.2.jar /jmx_prometheus_javaagent.jar
```

Comment:
Please do not fix it in this commit, but would it be possible to introduce a variable for this version number e.g similar like the ${REPO_URL} to ensure that next time we shall just update the prometheus agent only one place?